### PR TITLE
[Behat] Ensure AnnotationAgumentResolver as correct priority

### PR DESCRIPTION
**Pull request by miguelcleverti**
JIra: https://jira.ez.no/browse/EZP-25761

AnnotationArgumentResolver fails in php7 because it is being executed after Symfony2Extension's ArgumentResolver (which creates actual parameter instances).
The problem only happens in php7 because both services have the same priority and sort has a different behavior from php5.

The fix is to tag the service with a higher priority.

### CODE REVIEW

**Comment by joaoinacio** [comment]
+1

**Comment by bdunogier** [comment]
Typo in the commit message ("as" => "has"). But i'd rephrase it, we can use more info here:

What about

```
Fix EZP-25761: increase annotation argument resolver priority

It must run before the symfony behat extension ArgumentResolver.
```

**Comment by bdunogier** [comment]
@miguelcleverti I have edited my comment about 6.3. Could you please take care of that as well ?
